examples/Training/python/Tables.py

```diff
@@ -102,6 +102,17 @@
         print("   ", v)
 
 
+# Get data from every column of the specified rows using slice
+# ============================================================
+row_numbers = [3, 5, 7]
+print("\nGet All Data for rows with slice: ", row_numbers)
+data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))
```

Comment:
On the IDR
```py
In [24]: oid = 5223331

In [25]: open_table = s.sharedResources().openTable(omero.model.OriginalFileI(oi
    ...: d))

In [26]: row_numbers = [3, 5, 7]

In [27]: data = open_table.slice(range(len(open_table.getHeaders()), row_numbers
    ...: ))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-27-9685ca9bd501> in <module>
----> 1 data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))

TypeError: 'list' object cannot be interpreted as an integer
```

Reply:
Yes, I can repeat this with a different table on a different server.

```
In [10]: oid=98319

In [11]:

In [11]: open_table = session.sharedResources().openTable(omero.model.OriginalFi
    ...: leI(oid)
    ...: )

In [12]: row_numbers = [3, 5, 7]

In [13]: print("\nGet All Data for rows with slice: ", row_numbers)

Get All Data for rows with slice:  [3, 5, 7]

In [14]: data = open_table.slice(range(len(open_table.getHeaders()), row_numbers
    ...: ))
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-14-9685ca9bd501> in <module>
----> 1 data = open_table.slice(range(len(open_table.getHeaders()), row_numbers))

TypeError: 'list' object cannot be interpreted as an integer

In [15]:
```

Reply:
Also confirming that the corresponding non-slice example works

```
In [18]: data = open_table.readCoordinates(list(range(6)))
    ...: for col in data.columns:
    ...:     print("Data for Column: ", col.name)
    ...:     for v in col.values:
    ...:         print("   ", v)
    ...:
    ...:
Data for Column:  ImageNumber
    1.0
    1.0
    1.0
    1.0
...
```